### PR TITLE
[res] TensorFlowLiteRecipes for unnecessary Add removal

### PR DESCRIPTION
**Pull request by stamalakhov**
This commit adds models for testing unnecessary Add removal with various parameters.

Its correctness is tested at https://github.com/Samsung/ONE/pull/11770.
Draft: https://github.com/Samsung/ONE/pull/11770
Related: https://github.com/Samsung/ONE/issues/10358

ONE-DCO-1.0-Signed-off-by: s.malakhov <s.malakhov@partner.samsung.com>

### CODE REVIEW

**Review by seanshpark — APPROVED** [review]
LGTM

File: res/TensorFlowLiteRecipes/Net_Mul_Add_000/test.rule [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# To check if unnecessary Add is removed from sample graph
+
+RULE    "VERIFY_FILE_FORMAT"      $(verify_file_format) '=' 1
+
+RULE    "MUL_EXIST"             $(op_count MUL) '=' 1
+RULE    "ADD_NOT_EXIST"         $(op_count ADD) '=' 0
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/Net_Mul_Add_000/test.rule:6]: optional) would be better with align
```suggestion
RULE    "VERIFY_FILE_FORMAT"    $(verify_file_format) '=' 1

RULE    "MUL_EXIST"             $(op_count MUL) '=' 1
RULE    "ADD_NOT_EXIST"         $(op_count ADD) '=' 0
```

**Review by seanshpark — APPROVED** [review]
LGTM thank you!

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**